project/Dependencies.scala

```diff
@@ -5,5 +5,5 @@ object Dependencies {
   val scalatest     = "org.scalatest"       %% "scalatest"   % "3.2.9"
   val `cats-helper` = "com.evolutiongaming" %% "cats-helper" % "3.0.1"
   val random        = "com.evolutiongaming" %% "random"      % "1.0.0"
-  val `cats-effect` = "org.typelevel"       %% "cats-effect" % "3.3.4"
-}
\ No newline at end of file
+  val `cats-effect` = "org.typelevel"       %% "cats-effect" % "3.4.8"
```

Comment:
Is there a reason not to pull the version from cats-helper and have it separately?